src/sql/src/statements/transform/expand_interval.rs

```diff
@@ -36,7 +36,7 @@ lazy_static! {
         ("s","seconds"),
         ("millis","milliseconds"),
         ("mils","milliseconds"),
-        ("ms","microseconds"),
+        ("ms","milliseconds"),
         ("us","microseconds"),
         ("ns","nanoseconds"),
     ]);
```

Comment:
I wonder if the `INTERVAL_SHORT_NAME_MAPPING` should be renamed to `INTERVAL_ABBREVIATION_MAPPING`, WDYT?

Reply:
Sounds good!